### PR TITLE
Story/027 basic structure

### PR DESCRIPTION
**Pull request by e-mccoy**

### CODE REVIEW

**Review by ChrisThompsonK — APPROVED** [review]

File: Database-Files/CreateDatabase.sql [hunk 1]
```diff
@@ -0,0 +1,128 @@
+DROP SCHEMA IF EXISTS kJourneyDB;
```
Comment (JamesMatchett) [inline Database-Files/CreateDatabase.sql:1]: Can you attach a screenshot of this script running without errors?

File: Database-Files/dummy.sql [hunk 2]
```diff
@@ -0,0 +1,184 @@
+
+insert into capLead (name, photo, message)
```
Comment (JamesMatchett) [inline Database-Files/dummy.sql:2]: same with this script

File: ExpressAPI/db.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const mysql = require('mysql');
+
+const db = mysql.createConnection({
+    host: process.env.DB_HOST,
```
Comment (JamesMatchett) [inline ExpressAPI/db.js:4]: Do i get the env vars on my machine locally?, just not sure how to config this so it runs on my machine.

File: kjourney/.editorconfig [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```
Comment (Conormc98) [inline kjourney/.editorconfig:1]: Does this file need to be here?

File: Database-Files/CreateDatabase.sql (hunk 1 above)
Comment (ChrisThompsonK) [reply Database-Files/CreateDatabase.sql:1]: image

Comment (JamesMatchett) [reply Database-Files/CreateDatabase.sql:1]: looks good

File: kjourney/src/UI.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>kJourney</title>
+<link rel="icon" type="image/x-icon" href="favicon.ico">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<style>
```
Comment (Conormc98) [inline kjourney/src/UI.html:8]: Maybe put this in a css file?

File: Database-Files/dummy.sql (hunk 2 above)
Comment (ChrisThompsonK) [reply Database-Files/dummy.sql:2]: image
image

File: kjourney/src/app/app.component.css [hunk 6]
```diff
@@ -0,0 +1,99 @@
+<style>
```
Comment (Conormc98) [inline kjourney/src/app/app.component.css]: Does this css file need this <style> tag?

File: ExpressAPI/db.js (hunk 3 above)
Comment (ChrisThompsonK) [reply ExpressAPI/db.js:4]: Yes, you'll need to make a .env file with those values inside your ExpressAPI folder and add those values in as per your machine.

File: kjourney/src/app/app.component.html [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+<header>
+<h1></h1>
+<img src="logo.png" alt="kJourney Logo" display="inline-block" width="35%">
+</header>
+
+  <article>
+    <h1>Please enter your login details:</h1>
+    <h2>E-Mail:</h2>
+    <p><textarea cols="71" rows="1"></textarea></p>
```
Comment (Conormc98) [inline kjourney/src/app/app.component.html]: Any reason why we're using <textarea> over <input> ?

File: ExpressAPI/db.js (hunk 3 above)
Comment (JamesMatchett) [reply ExpressAPI/db.js:4]: thanks, that's all my comments

**Review by JamesMatchett — APPROVED** [review]

File: kjourney/src/app/app.component.css [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+  float: left;
+  width: 10%;
+  background: #00cbe4;
+  padding: 20px;
+  border: 2px solid black;
+  color: white;
+}
+
+nav ul {
+  overflow: scroll
```
Comment (Conormc98) [inline kjourney/src/app/app.component.css]: ;

File: kjourney/src/app/app.component.css (hunk 6 above)
Comment (Conormc98) [reply kjourney/src/app/app.component.css]: Remove the style tags as it will fail to compile

File: kjourney/src/app/app.component.html [hunk 9]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>kJourney</title>
+</title>
```
Comment (Conormc98) [inline kjourney/src/app/app.component.html]: Tries to close the title twice here

File: kjourney/src/app/app.component.html [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+  <article>
+    <h1>Please enter your login details:</h1>
+    <h2>E-Mail:</h2>
+    <p><textarea cols="71" rows="1"></textarea></p>
+    <h2>Password:</h2>
+    <p><textarea cols="71" rows="1"></textarea></p>
+    <p></p>
+    <input type="submit" value="Log-In" >
+  </article>
+</section>
```
Comment (Conormc98) [inline kjourney/src/app/app.component.html]: No section tag

**Review by Conormc98 — APPROVED** [review]

**Pull request merged**